charts/prometheus-mysql-exporter/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
```

Comment:
Do you have a plan to upgrade to v2?

Reply:
I think as soon as helm 2 deprecates?
Chart can still be used with helm 3 until then.

Updating now would mean no helm 2 support.
Do you have any reason to hurry?

Reply:
I agree it's best to wait until Nov 13. There is a way to start a chart API v2 version of a chart in the same repo sooner, but let's discuss that separately.